src/i_sound.h

```diff
@@ -51,22 +57,63 @@ void I_ShutdownSound(void);
 //  SFX I/O
 //
 
+extern int snd_resampler;
+extern int snd_module;
+extern boolean snd_hrtf;
+extern int snd_absorption;
+extern int snd_doppler;
+
+typedef struct sound_module_s
+{
+    boolean (*I_InitSound)(void);
+    boolean (*I_ReinitSound)(void);
+    boolean (*I_AllowReinitSound)(void);
```

Comment:
It's confusing that the struct members match the names of actual functions. Could you e.g. remove the `I_` prefix from these?

Reply:
Fixed by https://github.com/fabiangreffrath/woof/pull/1118/commits/894cbb0a2eb936db329568cc3a5f331aae89a143